### DISCUSSION TITLE
Regex and git history

### DISCUSSION THREAD

**Discussion by TH3xACE** [octokit/octokit.net#2678] [Q&A]
Hey guys,

First of all, this project is really nice, thanks for making it available.

I just started to look at the project and starting to use and was wondering whether it is possible to use regex for the search code ? or any other search ? Secondly is there a way to search also within git history? Commit ?

Thanks

**Comment by kfcampbell** [comment]
Hi! 

Unfortunately [GitHub's Search API](https://docs.github.com/en/rest/search?apiVersion=2022-11-28#about-search) does not support regular expressions. It [does support searching commits](https://docs.github.com/en/rest/search?apiVersion=2022-11-28#search-commits), which I _think_ is mapped to the SearchCode models in octokit.net.
